front/src/modules/rollingStock/components/RollingStockEditor/RollingStockEditorCurves.tsx

```diff
@@ -200,39 +208,43 @@ const RollingStockEditorCurves = ({
         }}
         selectedParamsSetter={updateSelectedParams}
       />
-      {selectedTractionMode && selectedCurve && selectedCurveIndex !== null && (
-        <div className="rollingstock-editor-curves d-flex p-3">
-          <CurveSpreadsheet
-            selectedCurve={selectedCurve}
-            selectedCurveIndex={selectedCurveIndex}
-            effortCurves={effortCurves}
-            selectedTractionMode={selectedTractionMode}
-            setEffortCurves={setEffortCurves}
-            isDefaultCurve={
-              selectedParams.comfortLevel === STANDARD_COMFORT_LEVEL &&
-              selectedParams.electricalProfile === null &&
-              selectedParams.powerRestriction === null
-            }
-          />
-          <div className="rollingstock-card-body">
-            {!isEmpty(curvesToDisplay[selectedTractionMode]?.curves) && (
-              <RollingStockCurve
-                curvesComfortList={[selectedParams.comfortLevel]}
-                data={curvesToDisplay}
-                isOnEditionMode
-                showPowerRestriction={showPowerRestriction}
-                hoveredElectricalParam={hoveredRollingstockParam}
-                selectedElectricalParam={
-                  showPowerRestriction
-                    ? selectedParams.powerRestriction
-                    : selectedParams.electricalProfile
-                }
-              />
-            )}
-            {children}
+      {selectedTractionMode &&
+        selectedCurve &&
+        selectedCurveIndex !== null &&
+        selectedTractionModeCurves !== null && (
```

Comment:
You don't need the null here
```suggestion
        selectedTractionModeCurves && (
```